### ISSUE TITLE
update images

### ISSUE THREAD

**Issue by jpetto** [mozmeao/lumbergh#458]
waiting on specifics from @gitbiz

**Comment by gitbiz** [comment]
Front page hero image: https://mozilla.app.box.com/file/491934571234

Jobs page hero image: https://drive.google.com/file/d/1RIZooXUnwbOS54nbWcIV9ZXYYyzj6APW/view

Jobs page (individual listings) hero image: https://mozilla.app.box.com/file/491942224967 

Internship page hero image: https://drive.google.com/drive/folders/19XyL7qOylUKVf32_M3A8tEpoTQX7fsir  (file is “USE THIS FOR CAREERS SITE.jpg”)

Benefits page hero image: https://mozilla.app.box.com/file/492067887529

Image on front page to replace the Ryan Baby one we’ll now use as Intern hero image: https://mozilla.app.box.com/file/491935509399

**Comment by jpetto** [comment]
@gitbiz can you put the box.com images in google drive? i do not have access to box. 😞

**Comment by gitbiz** [comment]
Front page hero image: https://drive.google.com/file/d/13aYsdIc0SPp1uIwXwyAIbq9XnyRwZSyZ/view?usp=sharing

Jobs page hero image: https://drive.google.com/file/d/1RIZooXUnwbOS54nbWcIV9ZXYYyzj6APW/view

Jobs page (individual listings) hero image: https://drive.google.com/file/d/1SLAATauDCBnt4jr2BnggkEQhOcOPnB9d/view?usp=sharing

Internship page hero image: https://drive.google.com/drive/folders/19XyL7qOylUKVf32_M3A8tEpoTQX7fsir (file is “USE THIS FOR CAREERS SITE.jpg”)

Benefits page hero image: https://drive.google.com/file/d/1FvBQCjElwUbvOpq6b2jcGf6d_Gl0-rIA/view?usp=sharing

Image on front page to replace the Ryan Baby one we’ll now use as Intern hero image: https://drive.google.com/file/d/1SiXUinankN3Glw1YsJUUL2mDjTwnAUWF/view?usp=sharing

**Comment by dianel** [comment]
not sure if we need a new ticket but we do need to replace the photo ono the jobs page https://careers.mozilla.org/listings/ to this photo: https://drive.google.com/file/d/1ZJtNiLSHcSr4vT0g1IFzM0XhCTDsJwCN/view?usp=sharing

or this photo: https://drive.google.com/file/d/1CYIjJEFXT8rj-tOjfi5qu-BLYyc_D6nn/view?usp=sharing

**Comment by dianel** [comment]
if you do finish on this one, hasan,....there's also this one :)  -> https://github.com/mozmeao/lumbergh/issues/540 (I can add a new image if we need it just lmk)

**Comment by dianel** [comment]
seeing if this is in dev now or still in process? -> https://github.com/mozmeao/lumbergh/issues/458#issuecomment-682105559

**Comment by helfi92** [comment]
It's currently waiting for review in https://github.com/mozmeao/lumbergh/pull/591
